node/src/service/service_parachain.rs

```diff
@@ -261,10 +261,6 @@ where
         bool,
     ) -> Result<Box<dyn ParachainConsensus<Block>>, sc_service::Error>,
 {
-    if matches!(parachain_config.role, Role::Light) {
-        return Err("Light client not supported!".into());
-    }
-
```

Comment:
Seems like a mistake here. Or is this by purpose?

Reply:
AFAIR Role::Light was removed.